File: prototype/ts/Point.ts
import { Model, model, prop } from 'mobx-keystone';

import { Point as IPoint } from './types';


@model('onto/Point')
class Point extends Model({
    x: prop<number>(0),
    y: prop<number>(0),
    z: prop<number>(0),
}) implements IPoint {}

export default Point;

File: packages/csharp/onto/OnToSharp/src/Types/Point.cs
using GraphQLCodeGen;


namespace Types {

    public class Point : Schema.Point {
        public int x { get; set; }
        public int y { get; set; }
        public int z { get; set; }

        public Point(int x = 0, int y = 0, int z = 0) {
            this.x = x;
            this.y = y;
            this.z = z;
        }
    }

}
